2024-01-17 15:30:18,914 INFO :: This message just shows basic information
2024-01-17 15:30:18,915 WARNING :: This message is about something you should pay attention to
2024-01-17 15:30:18,915 ERROR :: This message helps to debug an error that occurred in your program
2024-01-17 15:35:05,048 INFO :: This message just shows basic information
2024-01-17 15:35:05,049 WARNING :: This message is about something you should pay attention to
2024-01-17 15:35:05,049 ERROR :: This message helps to debug an error that occurred in your program
2024-01-17 15:35:05,049 INFO :: Hello from binance connector
2024-01-29 21:27:51,571 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:27:58,371 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -2014, 'msg': 'API-key format invalid.'} (error code 401)
2024-01-29 21:28:40,701 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:28:47,311 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-01-29 21:30:19,400 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:30:23,696 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-01-29 21:31:01,634 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:31:02,357 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-29 21:46:40,293 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:46:49,965 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-29 21:47:42,081 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:47:47,386 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-01-29 21:50:22,039 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:50:28,422 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-01-29 21:58:18,574 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:58:33,333 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:59:35,873 INFO :: Binance Futures Client successfully initialized
2024-01-29 21:59:43,710 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-01-29 22:04:12,833 INFO :: Binance Futures Client successfully initialized
2024-01-29 22:04:25,296 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-29 22:05:57,316 INFO :: Binance Futures Client successfully initialized
2024-01-29 22:06:20,270 INFO :: Binance Futures Client successfully initialized
2024-01-29 22:06:22,483 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 15:47:10,492 INFO :: Binance Futures Client successfully initialized
2024-01-30 15:47:12,282 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:20:58,212 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:21:00,965 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:21:47,178 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:21:50,466 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:22:17,507 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:22:19,886 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:22:21,938 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:25:58,352 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:26:02,819 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:52:51,217 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:52:56,223 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:59:26,505 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:59:32,196 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 17:59:44,589 INFO :: Binance Futures Client successfully initialized
2024-01-30 17:59:46,571 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 18:04:08,229 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:04:13,642 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': "Timestamp for this request was 1000ms ahead of the server's time."} (error code 400)
2024-01-30 18:05:02,793 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:05:08,519 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-01-30 18:38:31,935 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:39:25,084 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:39:28,080 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-01-30 18:43:36,647 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:43:45,229 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:43:46,104 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-01-30 18:44:14,523 INFO :: Binance Futures Client successfully initialized
2024-01-30 18:52:32,573 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:41:21,340 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:43:12,174 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:43:26,092 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:44:03,012 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:44:35,650 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:45:49,386 INFO :: Binance Futures Client successfully initialized
2024-02-01 19:49:24,475 INFO :: Binance Futures Client successfully initialized
2024-02-02 14:44:28,589 INFO :: Websocket connected
2024-02-02 14:44:28,589 INFO :: Binance connection opened
2024-02-02 14:44:38,882 ERROR :: Binance connection error: 
2024-02-02 14:44:39,430 ERROR :: error from callback <bound method BinanceFuturesClient.on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x1048a6f10>>: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-02 14:44:39,430 ERROR :: Binance connection error: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-02 14:44:39,430 INFO :: tearing down on exception 
2024-02-02 14:44:40,880 INFO :: Websocket connected
2024-02-02 14:44:40,880 INFO :: Binance connection opened
2024-02-02 14:45:03,700 INFO :: Websocket connected
2024-02-02 14:45:03,700 INFO :: Binance connection opened
2024-02-02 14:45:07,146 ERROR :: Binance connection error: 
2024-02-02 14:45:07,693 ERROR :: error from callback <bound method BinanceFuturesClient.on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x100be9750>>: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-02 14:45:07,693 ERROR :: Binance connection error: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-02 14:45:07,694 INFO :: tearing down on exception 
2024-02-02 14:45:09,161 INFO :: Websocket connected
2024-02-02 14:45:09,161 INFO :: Binance connection opened
2024-02-03 14:18:18,710 INFO :: Websocket connected
2024-02-03 14:18:18,712 INFO :: Binance connection opened
2024-02-03 14:19:23,384 ERROR :: Binance connection error: 
2024-02-03 14:19:23,938 ERROR :: error from callback <bound method BinanceFuturesClient.on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x1017c1250>>: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-03 14:19:23,939 ERROR :: Binance connection error: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-03 14:19:23,939 INFO :: tearing down on exception 
2024-02-03 14:19:25,686 INFO :: Websocket connected
2024-02-03 14:19:25,686 INFO :: Binance connection opened
2024-02-03 14:29:55,260 INFO :: Websocket connected
2024-02-03 14:29:55,262 INFO :: Binance connection opened
2024-02-03 14:30:09,559 ERROR :: Binance connection error: 
2024-02-03 14:30:10,071 ERROR :: error from callback <bound method BinanceFuturesClient.on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x10106d250>>: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-03 14:30:10,072 ERROR :: Binance connection error: BinanceFuturesClient.on_close() takes 2 positional arguments but 4 were given
2024-02-03 14:30:10,072 INFO :: tearing down on exception 
2024-02-03 14:30:11,951 INFO :: Websocket connected
2024-02-03 14:30:11,952 INFO :: Binance connection opened
2024-02-03 16:46:22,070 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-02-03 16:46:22,075 INFO :: Binance Futures Client successfully initialized
2024-02-03 16:46:24,102 INFO :: Websocket connected
2024-02-03 16:46:24,103 INFO :: Binance connection opened
2024-02-03 16:47:36,332 INFO :: Binance Futures Client successfully initialized
2024-02-03 16:47:38,326 INFO :: Websocket connected
2024-02-03 16:47:38,327 INFO :: Binance connection opened
2024-02-03 16:51:42,462 ERROR :: Connection error while making GET request to /fapi/v2/account: HTTPSConnectionPool(host='testnet.binancefuture.com', port=443): Max retries exceeded with url: /fapi/v2/account?timestamp=1706966502071&signature=57420312c890174d2d7677987849a47a72c0957f90a4059f8643ec05dadb7348 (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))
2024-02-03 16:51:42,465 INFO :: Binance Futures Client successfully initialized
2024-02-03 16:51:44,647 INFO :: Websocket connected
2024-02-03 16:51:44,648 INFO :: Binance connection opened
2024-02-03 16:53:46,891 INFO :: Binance Futures Client successfully initialized
2024-02-03 16:53:49,959 INFO :: Websocket connected
2024-02-03 16:53:49,959 INFO :: Binance connection opened
2024-02-06 20:30:02,621 INFO :: Binance Futures Client successfully initialized
2024-02-06 20:30:02,967 ERROR :: Connection error while making GET request to /api/v1/instrument/active: HTTPSConnectionPool(host='www.testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/instrument/active (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x102339090>: Failed to resolve 'www.testnet.bitmex.com' ([Errno 8] nodename nor servname provided, or not known)"))
2024-02-06 20:30:02,972 ERROR :: Connection error while making GET request to /api/v1/user/margin: HTTPSConnectionPool(host='www.testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/user/margin?currency=all (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10233b350>: Failed to resolve 'www.testnet.bitmex.com' ([Errno 8] nodename nor servname provided, or not known)"))
2024-02-06 20:30:02,972 INFO :: Bitmex Client successfully initialized
2024-02-06 20:30:10,275 INFO :: Websocket connected
2024-02-06 20:30:10,275 INFO :: Binance connection opened
2024-02-06 20:34:04,518 INFO :: Binance Futures Client successfully initialized
2024-02-06 20:34:04,522 ERROR :: Connection error while making GET request to /api/v1/instrument/active: HTTPSConnectionPool(host='www.testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/instrument/active (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x102252a10>: Failed to resolve 'www.testnet.bitmex.com' ([Errno 8] nodename nor servname provided, or not known)"))
2024-02-06 20:34:04,524 ERROR :: Connection error while making GET request to /api/v1/user/margin: HTTPSConnectionPool(host='www.testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/user/margin?currency=all (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1022530d0>: Failed to resolve 'www.testnet.bitmex.com' ([Errno 8] nodename nor servname provided, or not known)"))
2024-02-06 20:34:04,525 INFO :: Bitmex Client successfully initialized
2024-02-06 20:34:07,567 INFO :: Websocket connected
2024-02-06 20:34:07,567 INFO :: Binance connection opened
2024-02-06 20:35:47,924 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-06 20:35:47,928 ERROR :: Connection to remote host was lost. - goodbye
2024-02-06 20:35:47,929 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x100a08a90>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-06 20:35:47,929 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-06 20:35:55,455 INFO :: Websocket connected
2024-02-06 20:35:55,456 INFO :: Binance connection opened
2024-02-06 20:36:36,092 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-06 20:36:36,094 ERROR :: Connection to remote host was lost. - goodbye
2024-02-06 20:36:38,094 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:40,099 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:42,101 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:44,106 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:46,110 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:48,115 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:50,119 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:52,124 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:54,130 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:56,136 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:36:58,141 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:00,142 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:02,147 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:04,153 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:06,155 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:08,162 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:10,165 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:12,166 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:14,171 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:16,176 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:18,180 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:20,184 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:22,190 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:24,195 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:26,200 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:28,205 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:30,207 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:32,212 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:34,213 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:36,219 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:38,221 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:40,227 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:42,232 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:44,236 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:46,240 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:48,241 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:50,244 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:52,249 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:54,254 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:56,259 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:37:58,266 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:38:00,272 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:38:02,273 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:38:04,277 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:41:05,312 INFO :: Binance Futures Client successfully initialized
2024-02-06 20:41:05,316 ERROR :: Connection error while making GET request to /api/v1/instrument/active: HTTPSConnectionPool(host='www.testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/instrument/active (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1025891d0>: Failed to resolve 'www.testnet.bitmex.com' ([Errno 8] nodename nor servname provided, or not known)"))
2024-02-06 20:41:05,318 ERROR :: Connection error while making GET request to /api/v1/user/margin: HTTPSConnectionPool(host='www.testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/user/margin?currency=all (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10258b410>: Failed to resolve 'www.testnet.bitmex.com' ([Errno 8] nodename nor servname provided, or not known)"))
2024-02-06 20:41:05,319 INFO :: Bitmex Client successfully initialized
2024-02-06 20:41:08,544 INFO :: Websocket connected
2024-02-06 20:41:08,544 INFO :: Binance connection opened
2024-02-06 20:44:33,109 INFO :: Binance Futures Client successfully initialized
2024-02-06 20:44:34,877 ERROR :: Connection error while making GET request to /api/v1/instrument/active: HTTPSConnectionPool(host='testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/instrument/active (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))
2024-02-06 20:44:36,216 INFO :: Websocket connected
2024-02-06 20:44:36,217 INFO :: Binance connection opened
2024-02-06 20:44:37,011 INFO :: Bitmex Client successfully initialized
2024-02-06 20:46:16,598 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-06 20:46:16,601 ERROR :: Connection to remote host was lost. - goodbye
2024-02-06 20:46:16,602 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x104684a90>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-06 20:46:16,602 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-06 20:46:20,751 ERROR :: Binance connection error: [Errno 54] Connection reset by peer
2024-02-06 20:46:20,751 ERROR :: [Errno 54] Connection reset by peer - goodbye
2024-02-06 20:46:22,754 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:24,760 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:26,763 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:28,767 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:30,773 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:32,778 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:34,783 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:36,788 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:38,793 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:40,796 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:42,801 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:44,806 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:46,809 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:48,815 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:50,819 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:52,826 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:54,831 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:56,833 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:46:58,838 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 20:47:09,294 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-02-06 20:47:09,297 INFO :: Binance Futures Client successfully initialized
2024-02-06 20:47:11,309 INFO :: Websocket connected
2024-02-06 20:47:11,309 INFO :: Binance connection opened
2024-02-06 20:47:13,088 INFO :: Bitmex Client successfully initialized
2024-02-06 21:25:59,317 ERROR :: Connection error while making GET request to /fapi/v1/exchangeInfo: HTTPSConnectionPool(host='testnet.binancefuture.com', port=443): Max retries exceeded with url: /fapi/v1/exchangeInfo (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))
2024-02-06 21:25:59,998 ERROR :: Connection error while making GET request to /fapi/v1/account: ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2024-02-06 21:26:00,000 INFO :: Binance Futures Client successfully initialized
2024-02-06 21:26:03,841 INFO :: Bitmex Client successfully initialized
2024-02-06 21:26:05,683 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Order quantity must be a multiple of lot size: 100', 'name': 'HTTPError'}} (error code 400)
2024-02-06 21:26:07,663 INFO :: Websocket connected
2024-02-06 21:26:07,663 INFO :: Binance connection opened
2024-02-06 21:26:08,278 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x104b3c210>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-06 21:26:08,279 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-06 21:26:12,955 INFO :: Websocket connected
2024-02-06 21:26:12,957 INFO :: Binance connection opened
2024-02-06 21:26:17,307 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 21:26:19,310 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 21:26:21,311 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-06 21:27:04,083 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-02-06 21:27:04,088 INFO :: Binance Futures Client successfully initialized
2024-02-06 21:27:05,708 ERROR :: Connection error while making GET request to /api/v1/instrument/active: HTTPSConnectionPool(host='testnet.bitmex.com', port=443): Max retries exceeded with url: /api/v1/instrument/active (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))
2024-02-06 21:27:06,951 INFO :: Bitmex Client successfully initialized
2024-02-06 21:27:07,060 INFO :: Websocket connected
2024-02-06 21:27:07,061 INFO :: Binance connection opened
2024-02-08 09:56:59,857 ERROR :: Connection error while making GET request to /fapi/v1/account: HTTPSConnectionPool(host='testnet.binancefuture.com', port=443): Max retries exceeded with url: /fapi/v1/account?timestamp=1707373619555&signature=777439cf5cc7db926cddf261174d78bf55a5004f03eadad25b6adc279eefca31 (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))
2024-02-08 09:56:59,859 INFO :: Binance Futures Client successfully initialized
2024-02-08 09:57:01,570 INFO :: Websocket connected
2024-02-08 09:57:01,570 INFO :: Binance connection opened
2024-02-08 10:42:01,702 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -1021, 'msg': 'Timestamp for this request is outside of the recvWindow.'} (error code 400)
2024-02-08 10:42:01,706 INFO :: Binance Futures Client successfully initialized
2024-02-08 10:42:20,284 INFO :: Websocket connected
2024-02-08 10:42:20,286 INFO :: Binance connection opened
2024-02-08 10:42:21,821 ERROR :: Error while making GET request to /api/v1/instrument/active: {'error': {'message': 'This request has expired - `expires` is in the past. Current time: 1707376341', 'name': 'HTTPError'}} (error code 400)
2024-02-08 10:42:27,459 ERROR :: Error while making GET request to /api/v1/user/margin: {'error': {'message': 'This request has expired - `expires` is in the past. Current time: 1707376347', 'name': 'HTTPError'}} (error code 400)
2024-02-08 10:42:27,461 INFO :: Bitmex Client successfully initialized
2024-02-08 10:55:16,915 ERROR :: Connection error while making GET request to /fapi/v1/account: ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2024-02-08 10:55:16,918 INFO :: Binance Futures Client successfully initialized
2024-02-08 10:55:28,371 INFO :: Websocket connected
2024-02-08 10:55:28,373 INFO :: Binance connection opened
2024-02-08 10:55:35,530 ERROR :: Error while making GET request to /api/v1/instrument/active: {'error': {'message': 'This request has expired - `expires` is in the past. Current time: 1707377132', 'name': 'HTTPError'}} (error code 400)
2024-02-08 10:55:54,415 ERROR :: Connection error while making GET request to /api/v1/user/margin: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-02-08 10:55:54,417 INFO :: Bitmex Client successfully initialized
2024-02-08 10:59:03,342 ERROR :: Connection error while making GET request to /fapi/v1/exchangeInfo: HTTPSConnectionPool(host='testnet.binancefuture.com', port=443): Max retries exceeded with url: /fapi/v1/exchangeInfo (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1002)')))
2024-02-08 10:59:19,672 ERROR :: Connection error while making GET request to /fapi/v1/account: ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
2024-02-08 10:59:19,675 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:00:24,183 ERROR :: Connection error while making GET request to /api/v1/instrument/active: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-02-08 11:00:29,803 INFO :: Bitmex Client successfully initialized
2024-02-08 11:25:14,824 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-02-08 11:25:14,830 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:25:16,103 INFO :: Bitmex Client successfully initialized
2024-02-08 11:25:16,580 INFO :: Websocket connected
2024-02-08 11:25:16,581 INFO :: Binance connection opened
2024-02-08 11:25:16,651 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Order quantity must be a multiple of lot size: 100', 'name': 'HTTPError'}} (error code 400)
2024-02-08 11:28:04,300 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-02-08 11:28:04,304 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:28:05,857 INFO :: Bitmex Client successfully initialized
2024-02-08 11:28:06,068 INFO :: Websocket connected
2024-02-08 11:28:06,068 INFO :: Binance connection opened
2024-02-08 11:28:06,728 INFO :: Websocket connected
2024-02-08 11:28:06,728 INFO :: Bitmex connection opened
2024-02-08 11:29:59,084 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:30:00,372 INFO :: Bitmex Client successfully initialized
2024-02-08 11:30:00,952 INFO :: Websocket connected
2024-02-08 11:30:00,952 INFO :: Binance connection opened
2024-02-08 11:30:01,137 INFO :: Websocket connected
2024-02-08 11:30:01,137 INFO :: Bitmex connection opened
2024-02-08 11:34:14,752 ERROR :: Connection error while making GET request to /fapi/v1/exchangeInfo: HTTPSConnectionPool(host='testnet.binancefuture.com', port=443): Max retries exceeded with url: /fapi/v1/exchangeInfo (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x107017210>, 'Connection to testnet.binancefuture.com timed out. (connect timeout=None)'))
2024-02-08 11:38:16,495 ERROR :: Error while making GET request to /fapi/v1/account: {'code': -5000, 'msg': 'GET /fapi/v1/account is retired, please use GET /fapi/v2/account'} (error code 400)
2024-02-08 11:38:16,499 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:38:17,868 INFO :: Bitmex Client successfully initialized
2024-02-08 11:38:18,541 INFO :: Websocket connected
2024-02-08 11:38:18,541 INFO :: Binance connection opened
2024-02-08 11:38:18,723 INFO :: Websocket connected
2024-02-08 11:38:18,724 INFO :: Bitmex connection opened
2024-02-08 11:40:59,443 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:41:01,190 INFO :: Websocket connected
2024-02-08 11:41:01,190 INFO :: Binance connection opened
2024-02-08 11:42:50,991 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:42:52,773 INFO :: Websocket connected
2024-02-08 11:42:52,773 INFO :: Binance connection opened
2024-02-08 11:44:33,835 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:44:35,608 INFO :: Websocket connected
2024-02-08 11:44:35,608 INFO :: Binance connection opened
2024-02-08 11:44:35,695 INFO :: Bitmex Client successfully initialized
2024-02-08 11:44:36,378 INFO :: Websocket connected
2024-02-08 11:44:36,378 INFO :: Bitmex connection opened
2024-02-08 11:59:28,302 INFO :: Binance Futures Client successfully initialized
2024-02-08 11:59:30,844 INFO :: Websocket connected
2024-02-08 11:59:30,845 INFO :: Binance connection opened
2024-02-08 12:00:39,613 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:00:41,220 INFO :: Websocket connected
2024-02-08 12:00:41,221 INFO :: Binance connection opened
2024-02-08 12:01:23,472 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:01:25,103 INFO :: Websocket connected
2024-02-08 12:01:25,103 INFO :: Binance connection opened
2024-02-08 12:03:10,275 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:03:12,012 INFO :: Websocket connected
2024-02-08 12:03:12,013 INFO :: Binance connection opened
2024-02-08 12:03:12,564 INFO :: Bitmex Client successfully initialized
2024-02-08 12:03:13,289 INFO :: Websocket connected
2024-02-08 12:03:13,289 INFO :: Bitmex connection opened
2024-02-08 12:03:25,788 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:03:27,014 INFO :: Bitmex Client successfully initialized
2024-02-08 12:03:27,191 INFO :: Websocket connected
2024-02-08 12:03:27,191 INFO :: Binance connection opened
2024-02-08 12:03:27,803 INFO :: Websocket connected
2024-02-08 12:03:27,803 INFO :: Bitmex connection opened
2024-02-08 12:14:33,319 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:14:34,561 INFO :: Bitmex Client successfully initialized
2024-02-08 12:14:34,959 INFO :: Websocket connected
2024-02-08 12:14:34,959 INFO :: Binance connection opened
2024-02-08 12:48:03,724 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:48:05,125 INFO :: Bitmex Client successfully initialized
2024-02-08 12:48:05,398 INFO :: Websocket connected
2024-02-08 12:48:05,399 INFO :: Binance connection opened
2024-02-08 12:48:05,985 INFO :: Websocket connected
2024-02-08 12:48:05,986 INFO :: Bitmex connection opened
2024-02-08 12:49:14,582 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:49:15,904 INFO :: Bitmex Client successfully initialized
2024-02-08 12:49:16,086 INFO :: Websocket connected
2024-02-08 12:49:16,086 INFO :: Binance connection opened
2024-02-08 12:49:16,477 INFO :: Websocket connected
2024-02-08 12:49:16,477 INFO :: Bitmex connection opened
2024-02-08 12:51:56,921 INFO :: Binance Futures Client successfully initialized
2024-02-08 12:51:58,509 INFO :: Bitmex Client successfully initialized
2024-02-08 12:51:58,607 INFO :: Websocket connected
2024-02-08 12:51:58,608 INFO :: Binance connection opened
2024-02-08 12:51:59,156 INFO :: Websocket connected
2024-02-08 12:51:59,156 INFO :: Bitmex connection opened
2024-02-08 13:41:56,909 INFO :: Binance Futures Client successfully initialized
2024-02-08 13:41:58,378 INFO :: Bitmex Client successfully initialized
2024-02-08 13:41:58,660 INFO :: Websocket connected
2024-02-08 13:41:58,660 INFO :: Binance connection opened
2024-02-08 13:41:58,865 INFO :: Websocket connected
2024-02-08 13:41:58,866 INFO :: Bitmex connection opened
2024-02-08 13:42:08,887 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 13:42:08,889 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 13:42:08,889 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x101fae750>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 13:42:08,889 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 13:42:10,937 INFO :: Binance Futures Client successfully initialized
2024-02-08 13:42:11,891 INFO :: Bitmex Client successfully initialized
2024-02-08 13:42:12,467 INFO :: Websocket connected
2024-02-08 13:42:12,467 INFO :: Bitmex connection opened
2024-02-08 13:42:12,768 INFO :: Websocket connected
2024-02-08 13:42:12,768 INFO :: Binance connection opened
2024-02-08 13:42:22,771 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 13:42:22,772 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 13:42:22,772 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x101872750>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 13:42:22,772 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 13:42:26,333 INFO :: Websocket connected
2024-02-08 13:42:26,333 INFO :: Binance connection opened
2024-02-08 13:42:36,339 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 13:42:36,341 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 13:42:38,346 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:40,353 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:42,358 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:44,360 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:46,363 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:48,365 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:50,371 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:52,374 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:54,377 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:56,383 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:42:58,387 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:00,390 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:02,391 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:04,396 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:06,400 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:08,403 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:10,406 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:12,412 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:14,417 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:16,418 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:18,420 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:20,425 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:22,431 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:24,436 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:26,439 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:28,445 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:30,450 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:32,456 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:34,461 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:36,462 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:38,463 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:40,468 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:42,470 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:44,473 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:46,474 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:48,475 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:50,480 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:52,485 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:54,491 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:56,495 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:43:58,498 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:00,502 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:02,504 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:04,506 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:06,507 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:08,509 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:10,513 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:12,514 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:14,519 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:16,522 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:18,526 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:20,530 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:22,531 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:24,536 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:26,537 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:28,542 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:30,544 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:32,548 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:34,549 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:36,551 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:38,554 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:40,557 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:42,558 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:44,562 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:46,563 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:48,565 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:50,499 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:52,504 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:54,510 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:56,514 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:44:58,520 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:00,521 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:02,526 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:04,530 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:06,533 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:08,534 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:10,540 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:12,541 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:14,543 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:16,545 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:18,550 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:20,552 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:22,557 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:24,558 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:26,560 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:28,565 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:45:30,569 ERROR :: Binance error in run_forever() method: socket is already opened
2024-02-08 13:55:57,620 INFO :: Binance Futures Client successfully initialized
2024-02-08 13:55:59,231 INFO :: Bitmex Client successfully initialized
2024-02-08 13:55:59,476 INFO :: Websocket connected
2024-02-08 13:55:59,476 INFO :: Binance connection opened
2024-02-08 13:55:59,969 INFO :: Websocket connected
2024-02-08 13:55:59,970 INFO :: Bitmex connection opened
2024-02-08 13:56:09,483 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 13:56:09,483 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 13:56:09,484 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x1016aa750>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 13:56:09,484 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:39:28,027 INFO :: Binance Futures Client successfully initialized
2024-02-08 15:39:29,252 INFO :: Bitmex Client successfully initialized
2024-02-08 15:39:29,719 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Invalid orderQty', 'name': 'ValidationError'}} (error code 400)
2024-02-08 15:39:29,985 INFO :: Websocket connected
2024-02-08 15:39:29,985 INFO :: Binance connection opened
2024-02-08 15:39:29,986 INFO :: Websocket connected
2024-02-08 15:39:29,986 INFO :: Bitmex connection opened
2024-02-08 15:39:55,481 INFO :: Binance Futures Client successfully initialized
2024-02-08 15:39:56,560 INFO :: Bitmex Client successfully initialized
2024-02-08 15:39:57,040 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Invalid orderQty', 'name': 'ValidationError'}} (error code 400)
2024-02-08 15:39:57,041 INFO :: Websocket connected
2024-02-08 15:39:57,249 INFO :: Binance connection opened
2024-02-08 15:39:57,251 INFO :: Websocket connected
2024-02-08 15:39:57,251 INFO :: Bitmex connection opened
2024-02-08 15:40:07,255 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 15:40:07,257 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 15:40:07,257 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x10310f1d0>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:40:07,257 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:40:10,905 INFO :: Websocket connected
2024-02-08 15:40:10,905 INFO :: Binance connection opened
2024-02-08 15:41:02,818 INFO :: Binance Futures Client successfully initialized
2024-02-08 15:41:04,410 INFO :: Websocket connected
2024-02-08 15:41:04,410 INFO :: Binance connection opened
2024-02-08 15:41:04,529 INFO :: Bitmex Client successfully initialized
2024-02-08 15:41:05,080 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Invalid orderQty', 'name': 'ValidationError'}} (error code 400)
2024-02-08 15:41:05,137 INFO :: Websocket connected
2024-02-08 15:41:05,137 INFO :: Bitmex connection opened
2024-02-08 15:41:14,413 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 15:41:14,414 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 15:41:14,414 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x10269cad0>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:41:14,415 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:41:18,059 INFO :: Websocket connected
2024-02-08 15:41:18,059 INFO :: Binance connection opened
2024-02-08 15:41:28,062 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 15:41:28,063 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 15:41:31,313 INFO :: Binance Futures Client successfully initialized
2024-02-08 15:41:32,418 INFO :: Bitmex Client successfully initialized
2024-02-08 15:41:32,831 INFO :: Websocket connected
2024-02-08 15:41:32,831 INFO :: Binance connection opened
2024-02-08 15:41:32,946 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Invalid orderQty', 'name': 'ValidationError'}} (error code 400)
2024-02-08 15:41:33,156 INFO :: Websocket connected
2024-02-08 15:41:33,156 INFO :: Bitmex connection opened
2024-02-08 15:41:42,840 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-08 15:41:42,842 ERROR :: Connection to remote host was lost. - goodbye
2024-02-08 15:41:42,842 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x102797210>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:41:42,843 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-08 15:41:46,403 INFO :: Websocket connected
2024-02-08 15:41:46,404 INFO :: Binance connection opened
2024-02-08 15:53:35,201 INFO :: Binance Futures Client successfully initialized
2024-02-08 15:53:36,760 INFO :: Bitmex Client successfully initialized
2024-02-08 15:53:36,946 INFO :: Websocket connected
2024-02-08 15:53:36,947 INFO :: Binance connection opened
2024-02-08 15:53:37,276 ERROR :: Error while making POST request to /api/v1/order: {'error': {'message': 'Invalid orderQty', 'name': 'ValidationError'}} (error code 400)
2024-02-08 15:53:37,535 INFO :: Websocket connected
2024-02-08 15:53:37,536 INFO :: Bitmex connection opened
2024-02-11 20:31:13,022 INFO :: Binance Futures Client successfully initialized
2024-02-11 20:31:14,404 INFO :: Bitmex Client successfully initialized
2024-02-11 20:31:14,933 INFO :: Websocket connected
2024-02-11 20:31:14,933 INFO :: Binance connection opened
2024-02-11 20:31:15,500 INFO :: Websocket connected
2024-02-11 20:31:15,500 INFO :: Bitmex connection opened
2024-02-11 20:32:02,635 INFO :: Binance Futures Client successfully initialized
2024-02-11 20:32:04,209 INFO :: Websocket connected
2024-02-11 20:32:04,209 INFO :: Binance connection opened
2024-02-11 20:32:04,259 INFO :: Bitmex Client successfully initialized
2024-02-11 20:32:05,442 INFO :: Websocket connected
2024-02-11 20:32:05,442 INFO :: Bitmex connection opened
2024-02-11 21:34:18,134 INFO :: Binance Futures Client successfully initialized
2024-02-11 21:34:20,056 INFO :: Bitmex Client successfully initialized
2024-02-11 21:34:20,887 INFO :: Websocket connected
2024-02-11 21:34:20,888 INFO :: Binance connection opened
2024-02-11 21:34:20,890 INFO :: Websocket connected
2024-02-11 21:34:20,890 INFO :: Bitmex connection opened
2024-02-11 21:34:30,891 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-11 21:34:30,893 ERROR :: Connection to remote host was lost. - goodbye
2024-02-11 21:34:30,893 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x10430c850>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:34:30,893 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:34:34,632 INFO :: Websocket connected
2024-02-11 21:34:34,632 INFO :: Binance connection opened
2024-02-11 21:43:25,045 INFO :: Binance Futures Client successfully initialized
2024-02-11 21:43:26,438 INFO :: Bitmex Client successfully initialized
2024-02-11 21:43:26,667 INFO :: Websocket connected
2024-02-11 21:43:26,667 INFO :: Binance connection opened
2024-02-11 21:43:28,059 INFO :: Websocket connected
2024-02-11 21:43:28,060 INFO :: Bitmex connection opened
2024-02-11 21:43:36,673 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-11 21:43:36,673 ERROR :: Connection to remote host was lost. - goodbye
2024-02-11 21:43:36,674 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x106010890>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:43:36,674 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:44:32,022 INFO :: Binance Futures Client successfully initialized
2024-02-11 21:44:33,755 INFO :: Websocket connected
2024-02-11 21:44:33,756 INFO :: Binance connection opened
2024-02-11 21:44:37,948 INFO :: Bitmex Client successfully initialized
2024-02-11 21:44:38,737 INFO :: Websocket connected
2024-02-11 21:44:38,737 INFO :: Bitmex connection opened
2024-02-11 21:44:43,759 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-11 21:44:43,759 ERROR :: Connection to remote host was lost. - goodbye
2024-02-11 21:44:43,760 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x1062c4f10>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:44:43,760 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:48:23,512 INFO :: Binance Futures Client successfully initialized
2024-02-11 21:48:25,051 INFO :: Bitmex Client successfully initialized
2024-02-11 21:48:25,859 INFO :: Websocket connected
2024-02-11 21:48:25,859 INFO :: Binance connection opened
2024-02-11 21:48:26,083 INFO :: Websocket connected
2024-02-11 21:48:26,084 INFO :: Bitmex connection opened
2024-02-11 21:48:35,863 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-11 21:48:35,865 ERROR :: Connection to remote host was lost. - goodbye
2024-02-11 21:48:35,866 ERROR :: error from callback <bound method BinanceFuturesClient._on_close of <tradingBot.connectors.binance_futures.BinanceFuturesClient object at 0x104e58410>>: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:48:35,866 ERROR :: Binance connection error: BinanceFuturesClient._on_close() takes 2 positional arguments but 4 were given
2024-02-11 21:48:39,995 INFO :: Websocket connected
2024-02-11 21:48:39,995 INFO :: Binance connection opened
2024-02-11 21:48:50,112 ERROR :: Binance connection error: Connection to remote host was lost.
2024-02-11 21:48:50,114 ERROR :: Connection to remote host was lost. - goodbye
